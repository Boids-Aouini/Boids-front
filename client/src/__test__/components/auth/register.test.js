import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import configureStore from 'redux-mock-store';
import Register from '../../../components/auth/register';

const storeMock = configureStore();
let intialTestState = {
    auth: {
        openedAcc: null
    }
}
let store = storeMock(intialTestState)

describe('<Register />', () => {
    describe('render()', () => {
        test('render without crashing', () => {
            shallow(<Register store={store} />)
        })
        test('renders component', () => {
            const wrapper = shallow(<Register store={store} />);
            const component = wrapper.dive();

            expect(toJson(component)).toMatchSnapshot();
        })

    })
})