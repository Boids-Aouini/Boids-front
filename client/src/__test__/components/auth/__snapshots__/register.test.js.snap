// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Register /> render() renders component 1`] = `
<Register
  auth={
    Object {
      "openedAcc": null,
    }
  }
  registerAction={[Function]}
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
/>
`;
